edx_when/models.py

```diff
@@ -84,6 +84,16 @@ def clean(self):
             raise ValidationError(_("Absolute and relative dates cannot both be used"))
 
 
+class CourseVersion(models.Model):
```

Comment:
Why have this as its own model rather than just a field on a ContentDate?

Reply:
Good question! Because I just can't help myself from normalizing DB data? 😸 It would also work fine as a field on `ContentDate` - I arbitrarily chose to use a new model.